File: PHP/Command/SimpleCommand.php
<?php
class SimpleCommand implements Command
{
    private $payload;

    public function __construct(string $payload)
    {
        $this->payload = $payload;
    }

    public function execute(): void
    {
        echo "SimpleCommand: does the request  (" . $this->payload . ")<br/>";
    }
}

File: Csharp/Command/Command/SimpleCommand.cs
using System;

namespace TheRayCode.Command
{
    class SimpleCommand : ICommand
    {
        private string _payload = string.Empty;

        public SimpleCommand(string payload)
        {
            this._payload = payload;
        }

        public void Execute()
        {
            Console.WriteLine($"SimpleCommand: does the request ({this._payload})");
        }
    }
}